### ISSUE TITLE
What can be improved from M1?

### ISSUE THREAD

**Issue by afiqdanialll** [afiqdanialll/INF2001-P8-4#35]
To be done after M1 submission. Comment down below.

**Comment by symphonyalpha** [comment]
Probably better time management so we don't cram everything in the last week

**Comment by afiqdanialll** [comment]
I agree, we could be better in our time management and be more communicative when needed.

**Comment by zhiyinang** [comment]
Time management, replying to messages in group chats when needed, and assigning the tasks evenly so that nobody will feel stressed out in completing the task/ tasks.

**Comment by ivanlsy** [comment]
While time management could be improved as mentioned by other team members, but we can take this as a learning curve on how in projects, there will always be room for improvement/areas of adjustments (minor ones) towards the end of milestones. We should focus more on setting clear deadlines and following up on progress of completion for each task. 

Despite these flaws, the team still managed to complete required tasks. The teamwork for the upcoming milestone will certainly improve. 

Cheers to the team.

**Comment by Ruizheee** [comment]
Time management could definitely have some room for improvement, in the subsequent milestones, we should better organize team meetings to allow for better communication

**Comment by juunweiii** [comment]
I would say time management too, the task were mainly completed only after the consultation, which is around 1-2 week prior to submission, which the timeline could have been better managed. However, still thanks to the teams for persevering on and not giving up despite rushing for time.
